docs/api/symbol-package-publish-resource.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Push and Delete, NuGet API | Microsoft Docs
+author:
+- cristinamanu
+- kraigb
+ms.author:
+- cristinamanu
+- kraigb
+manager: skofman
+ms.date: 10/30/2018
+ms.topic: reference
+ms.prod: nuget
+ms.technology: null
+description: The publish service allows clients to publish new symbol packages.
+keywords: NuGet API push symbol package
+ms.reviewer:
+- karann
```

Comment:
You don't need a YAML array here. `ms.reviewer: karann` is fine.

Reply:
Updated.